charts/sonarqube-dce/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [5.1.2]
+* Refactor Pdb and Ingress to be compatible with static compatibitly test and 1.19 minimum requirement
```

Comment:
I would raise one point about the 1.19 minimum requirement listing all the apiVersion that we do not support anymore as result of this (e.g., a few in the ingress, see below).

Reply:
We probably dont have too, because our old if/else with 1.19 as a requirement were always returning true.

So if i am not missing a case people was already having the latest ApiVersion in all cases.

Based on that i would keep it that ways, what do you guys think ?

Reply:
fair point, we can actually avoid mentioning this in the changelog.